packages/grid/src/index.ts

```diff
@@ -6,16 +6,21 @@ export * from './GridMetricCalculator';
 export * from './GridMetrics';
 export { default as GridModel } from './GridModel';
 export { default as GridMouseHandler } from './GridMouseHandler';
+export * from './GridMouseHandler';
 export { default as GridRange } from './GridRange';
+export * from './GridRange';
 export { default as GridRenderer } from './GridRenderer';
+export * from './GridRenderer';
 export { default as GridTestUtils } from './GridTestUtils';
 export { default as GridTheme } from './GridTheme';
 export type { GridTheme as GridThemeType } from './GridTheme';
 export { default as GridUtils } from './GridUtils';
+export * from './GridUtils';
```

Comment:
Could change GridUtils to `export class GridUtils` and avoid the default import here